recipe/recipe.yaml

```
@@ -18,18 +19,21 @@ build:
 
 requirements:
   host:
-    - python
+    - python ${{ python_min }}
     - pip
     - setuptools
   run:
-    - python
+    - python >=${{ python_min }}
     - numba >=0.59.1
 
 tests:
-  - python:
-      imports:
-      - numba_cuda
-      pip_check: true
+  - requirements:
+      run:
+        - python ${{ python_min }}
+        - pip
+  - script:
+      - python -m pip check
+      - python -c "import numba_cuda"
```

Missed this was [one list item]( https://github.com/conda/ceps/blob/4f4ac7545d4ba42fb10f71e40ad66ec865daab2e/cep-0014.md?plain=1#L801-L805 ) 🤦‍♂️

```suggestion
  - requirements:
      run:
        - python ${{ python_min }}
        - pip
    script:
      - python -m pip check
      - python -c "import numba_cuda"
```